searches/unbounded_binary_search.php

```diff
@@ -0,0 +1,71 @@
+<?php
+// PHP program for Binary Search
+ 
+// Let's take an example function
+// as f(x) = x^2 - 10*x - 20
+// Note that f(x) can be any
+// monotonically increasing function
+function f($x)
+{
+    return ($x * $x - 10 * $x - 20);
+}
+ 
+// Returns the value x where above
+// function f() becomes positive
+// first time.
+function findFirstPositive()
+{
+    // When first value
+    // itself is positive
+    if (f(0) > 0)
+        return 0;
+ 
+    // Find 'high' for binary
+    // search by repeated doubling
+    $i = 1;
+    while (f($i) <= 0)
+        $i = $i * 2;
+ 
+    // Call binary search
+    return binarySearch(intval($i / 2), $i);
+}
+ 
+// Searches first positive value
+// of f(i) where low <= i <= high
+function binarySearch($low, $high)
```

Comment:
Since we already have a `binarySearch` function in the `Searches` file directory, you'll need to either change the name of this function or put it into a class